{{cookiecutter.project_name}}/{{cookiecutter.project_slug}}/settings/base.py

```diff
@@ -351,6 +352,7 @@
     ('a4comments', 'comment'),
     ('{{ cookiecutter.project_app_prefix }}_ideas', 'idea'),
     ('{{ cookiecutter.project_app_prefix }}_mapideas', 'mapidea'),
+    ('{{ cookiecutter.project_app_prefix }}polls', 'poll')
```

Comment:
I removed this as it wouldn't install, I think because path is wrong which i'll look into but I was just wondering should polls be commentable anyway? (if that's what this is doing?) @fuzzylogic2000 @MagdaN